docs/input/docs/getting-started/whymaybe.md

```diff
@@ -0,0 +1,30 @@
+---
+Order: 10
+Title: Why does BBT.Maybe exist?
+Description: Introduction to BBT.Maybe, what problems are solved and how it is distinctive from other implementations of functional option type.
+---
+
+## Make optional reference type values explicit
+
+BBT.Maybe is an implementation of the functional option type pattern.
+
+BBT.Maybe has the following goals:
+
+* Make nullable references explicit
+* Prevent null reference exceptions on reference calls
+* Provide an option type with broad usage over all layers of code
+
+Numerous implementations of functional option type already exist. In focus of this realization are:
+
+* Prevent access to null value
+* Implementation of maybe for both optional reference and nullable value types
+* Preventing of bloated functionality
+* Usage of maybe in communication layer
+
+Implementation details of maybe are:
```

Comment:
_Implementation_ details shouldn't go into the _Why_ section -> Rather _Technical core principles_

Reply:
Moved implementation details to principles section.